app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -303,6 +361,7 @@ private void hideInfoLayoutArcPointer() {
 
     @Override
     public void onDestroy() {
+        timer.cancel();
         super.onDestroy();
```

Comment:
Agreed. But separate commit to fix that. I added timer.cancel() first because I'm trying to avoid timer trying to grab the screen while we exit() the ScreenGrabber.